build-vm-nspawn

```diff
@@ -0,0 +1,74 @@
+#
+# Docker specific functions
```

Comment:
This should be `nspawn`

Reply:
Oops, missed this one. I was using `build-vm-docker` as a template. Fixed.